source/vir/src/sst_to_air.rs

```diff
@@ -1710,9 +1712,11 @@ pub fn body_stm_to_air(
             ]
         } else if is_bit_vector_mode {
             vec![
+                mk_option_command("sat.euf", "true"),
```

Comment:
I had forgotten one thing: can you collect this options somewhere? (They currently are duplicated for assert_by_bitvector and #[verifier(bit_vector)]